### PR TITLE
Added code for ctags provider

### PR DESCRIPTION
**Pull request by shubhaangi04**

### CODE REVIEW

File: sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/provider/CTAGSProvider.java [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
     }
+    
+    /**
+     * To checkout ctags in the temp folder.
+     * @param cTagsURL : URL to clone CTags
+     * @param checkoutPath : Path in which the CTags folder will be checked out
+     */
+    public static void checkoutCTags(String cTagsURL, String checkoutPath) {
+      try {
+        Git.cloneRepository()
```
Comment (krvikas1011) [inline sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/provider/CTAGSProvider.java:25]: Extract this to a class

File: sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/util/FileOpertionsHandler.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+/**
+ * To handle the file operations.
+ *
+ */
+public class FileOpertionsHandler {
+	/**
+     * Get temporary directory/folder.
+     * @return the path of the temporary folder
+     */
+    public static String getTempUserDirectory() {
```
Comment (krvikas1011) [inline sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/util/FileOpertionsHandler.java:14]: Can this be renamed.

File: sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/util/FileOpertionsHandler.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    	return System.getProperty("java.io.tmpdir");
+    }
+    
+    /**
+     * Create a folder inside the temporary directory.
+     * @return path of the new folder 
+     */
+    public static String createFolderInTempDirectory() {
+    	String path = getTempUserDirectory() + "/Method_Parser_CTags";
```
Comment (krvikas1011) [inline sandbox/tags-manager/src/main/java/com/vikas/method_parser/tags_manager/util/FileOpertionsHandler.java:23]: Please de-couple

**Review by thisissvikas — CHANGES_REQUESTED** [review]

**Comment by krvikas1011** [comment]
Please rename this PR to what it actually is.

**Pull request closed without merging**